lib/Backends/Interpreter/InterpreterDeviceManager.h

```diff
@@ -17,14 +17,23 @@
 #define GLOW_BACKENDS_INTERPRETER_INTERPRETERDEVICEMANAGER_H
 
 #include "glow/Backends/QueueBackedDeviceManager.h"
+#include <future>
 
 namespace glow {
+
+using TensorPipeTy = std::pair<std::future<Tensor>, std::promise<Tensor>>;
```

Comment:
Consider replace std::pair with custom struct, it's very possible to add new field in the future.